modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Layout/Header/HeaderTabs.tsx

```diff
@@ -14,30 +14,35 @@
 
 import ClayTabs from '@clayui/tabs';
 import {useAtom} from 'jotai';
-import {useNavigate} from 'react-router-dom';
+import {useLocation, useNavigate} from 'react-router-dom';
 import {headerAtom} from '~/atoms';
 
 const HeaderTabs = () => {
 	const [tabs] = useAtom(headerAtom.tabs);
 
+	const {pathname} = useLocation();
+	const isEditCase = pathname.includes('edit' || 'complete');
```

Comment:
This implementation doesn´t work correctly, because the includes always check the first, and the first is always truthy,  you can you an array to check this, verifying each index of this array.